etstool.py

```diff
@@ -180,6 +216,7 @@ def test_clean(runtime):
     try:
         install(args=args, standalone_mode=False)
         test(args=args, standalone_mode=False)
+        docs(args=args, standalone_mode=False)
```

Comment:
If we're running the `docs` build here, should we be using `install_docs` rather than `install`?

Reply:
This is (almost unintentionally) fixed by the changes to install